addons/calendar/models/calendar.py

```diff
@@ -353,8 +353,13 @@ def do_check_alarm_for_one_date(self, one_date, event, event_maxdelta, in_the_ne
 
     @api.model
     def get_next_mail(self):
-        now = fields.Datetime.to_string(fields.Datetime.now())
-        last_notif_mail = self.env['ir.config_parameter'].sudo().get_param('calendar.last_notif_mail', default=now)
+        if self.env.context.get('lastcall'):
+            last_notif_mail = fields.Datetime.to_string(self.env.context['lastcall'])
+        else:
+            last_notif_mail = self.env['ir.config_parameter'].sudo().get_param(
```

Comment:
Do you think this backward-compatibility is worth it?
I would remove that part and use `self.env.context.get('lastcall') or fields.Datetime.now()` in the expression above.
We can leave a migration note that says: set the `lastcall` of the `ir.cron` to the value of the `ir.config_parameter`.

Reply:
@rco-odoo removed the BC bit